util/process_wrapper/options.rs

```diff
@@ -63,6 +64,7 @@ pub(crate) fn options() -> Result<Options, OptionError> {
     let mut rustc_output_format_raw = None;
     let mut flags = Flags::new();
     flags.define_repeated_flag("--subst", "", &mut subst_mapping_raw);
+    flags.define_flag("--subst-multipwd", "", &mut subst_multipwd);
```

Comment:
Can you explain the thought behind the name `multipwd` and why an additional `--subst` flag would not be sufficient?

Reply:
I don't have strong opinions on the name, open to alternatives.

I don't see how to go from `a b c` to `somewhere/a somewhere/b somewhere/c` with a single substitution. It has to operate on each of the paths separately, and Starlark can't see each path separately without fully reimplementing `$(locations)` expansion.

Reply:
The idea that there's multiple "present working directories" I think is what makes me defensive of the change. Why not pass multiple `--subst` values?

Reply:
Ah, I see the confusion from the name: it's a single working directory, substituted into multiple paths.